src/api/install.cpp

```diff
@@ -53,8 +53,6 @@ namespace mamba
 
         reproc::options options;
         options.redirect.parent = true;
-        std::string cwd = ctx.target_prefix;
-        options.working_directory = cwd.c_str();
 
```

Comment:
I think it would still be better to explicitly set the current work directory (as they do in the conda code I linked).

Reply:
Right, currently I have a hard time finding a good spot to set the cwd option. We need to set it in the config instance, when the environment.yaml file is being parsed.